carousel/docs/tutorials/tutorial_2.rst

```diff
@@ -138,32 +138,38 @@ Calculations can also be specified in a parameter file. For example copy the
 following into ``PVPower/calculations/utils.json``::
 
     {
-      "dependencies": ["PerformanceCalcs"],
-      "static": [
-        {
+      energy = {
```

Comment:
[There's an example in PVPower of what the parameter file for the utility calculations *should* look like.](../../../blob/master/examples/PVPower/calculations/utils.json):

```python
{
  "energy": {
    "is_dynamic": false,
    "dependencies": ["ac_power", "daterange"],
    "formula": "f_energy",
    "args": {
      "outputs": {"ac_power": "Pac", "times": "timestamps"}
    },
    "returns": ["hourly_energy", "hourly_timeseries"]
  },
  "monthly_rollup": {
    "is_dynamic": false,
    "dependencies": ["energy"],
    "formula": "f_rollup",
    "args": {
      "data": {"freq": "MONTHLY"},
      "outputs": {"items": "hourly_energy", "times": "hourly_timeseries"}
    },
    "returns": ["monthly_energy"]
  },
  "yearly_rollup": {
    "is_dynamic": false,
    "dependencies": ["energy"],
    "formula": "f_rollup",
    "args": {
      "data": {"freq": "YEARLY"},
      "outputs": {"items": "hourly_energy", "times": "hourly_timeseries"}
    },
    "returns": ["annual_energy"]
  }
}
```